typings/index.d.ts

```diff
@@ -275,7 +275,7 @@ declare namespace local {
      *
      * @param {(str: string) => string} [cb]
      */
-    help(cb?: (str: string) => string): void;
+    help(cb?: (str: string) => string): never;
```

Comment:
@blair `never` type is incorrect for help function, it doesn't throw or failing to terminate.
https://blog.mariusschulz.com/2016/11/18/typescript-2-0-the-never-type

Reply:
@vanesyan According to https://github.com/tj/commander.js/#helpcb the method "Output help information and exit immediately" so it terminates the process.

Reply:
Sorry, you're correct. `help` terminates process.